packages/perseus/src/__tests__/test-utils.test.ts

```diff
@@ -0,0 +1,33 @@
+import {describe, it, expect} from "@jest/globals";
+
+import {generateTestPerseusItem} from "../util/test-utils";
+
+import {
+    basicObject,
+    expected1,
+    questionCustomInfo,
+    expected2,
+    answerAreaCustomInfo,
+    hintsCustomInfo,
+    expected3,
+} from "./test-items/test-utils-items";
+
+describe("generateTestPerseusItem", () => {
```

Comment:
Would love suggestions on ways to make these tests more useful or additional tests that could be created for this change. :)